src/main/java/org/folio/search/service/consortium/ConsortiumInstanceService.java

```diff
@@ -164,4 +190,17 @@ private boolean isCentralTenant(String tenantId) {
     var centralTenant = consortiumTenantService.getCentralTenant(tenantId);
     return centralTenant.isPresent() && centralTenant.get().equals(tenantId);
   }
+
+  private String getCentralId() {
+    return consortiumTenantService.getCentralTenant(context.getTenantId()).orElse(null);
```

Comment:
Shouldn't we just use 'TenantProvider' here, which returns initially passed tenantId in case central is absent?
Or is there some reason for passing 'null'?